lib/src/week_view/_internal_week_view_page.dart

```diff
@@ -104,15 +105,11 @@ class InternalWeekViewPage<T> extends StatelessWidget {
       height: height + weekTitleHeight,
       width: width,
       child: Column(
-        mainAxisSize: MainAxisSize.max,
-        mainAxisAlignment: MainAxisAlignment.start,
         crossAxisAlignment: CrossAxisAlignment.end,
         children: [
           SizedBox(
             width: width,
             child: Row(
-              mainAxisSize: MainAxisSize.max,
-              mainAxisAlignment: MainAxisAlignment.start,
               crossAxisAlignment: CrossAxisAlignment.start,
```

Comment:
Why we removed `MainAxisSize` and `MainAxisAlignment`?

Reply:
Values we provided as arguments are default values for Row. So whether we provide it or not doesn't matter.